packages/app/src/specs/SpecsList.vue

```diff
@@ -213,27 +400,56 @@ function handleClear () {
 }
 
 const specs = computed(() => {
-  const specs = cachedSpecs.value.map((x) => makeFuzzyFoundSpec(x))
+  const fuzzyFoundSpecs = cachedSpecs.value.map((x) => {
+    const s = makeFuzzyFoundSpec(x)
+
+    return s
+  })
 
   if (!debouncedSearchString.value) {
-    return specs
+    return fuzzyFoundSpecs
   }
 
-  return fuzzySortSpecs(specs, debouncedSearchString.value)
+  return fuzzySortSpecs(fuzzyFoundSpecs, debouncedSearchString.value)
 })
 
+// Maintain a cache of what tree directories are expanded/collapsed so the tree state is visually preserved
```

Comment:
Absolutely agree that `SpecsList` has become unwieldy. I think we could at a minimum split this into a management component to maintain the overall data and interactions, and a second to handle the display aspects with the virtualized list and such. Not something I'm eager to do at this moment but I'd love to do this as a cleanup item.

Reply:
We should definitely have a chat soon about the future of the specs list, I think we can simplify the specs list and the search functionality a whole lot and drop virtualization. I'll put something on the calendar for after the break and invite some the 4 of us, we can add more people if needed.

Reply:
Agreed. Scrolling past more than a couple of screens to find a spec is not something I think most people will really do. I think they'll type the search term. Talked some with Ryan about the possible future implementations around this today, since we were talking anyway and it sounds like we are pretty open to different solutions around this list.